packages/web/src/store/application/ui/buy-audio/sagas.ts

```diff
@@ -431,20 +413,20 @@ function* getAudioPurchaseInfo({
       'finalized'
     )
 
-    const estimatedLamports = BN.max(
-      new BN(inSol)
-        .add(new BN(associatedAccountCreationFees))
-        .add(new BN(transactionFees))
-        .add(new BN(rootAccountMinBalance))
-        .sub(new BN(existingBalance)),
-      new BN(0)
+    const estimatedLamports = Math.max(
+      inSol +
+        associatedAccountCreationFees +
+        transactionFees +
+        rootAccountMinBalance +
+        existingBalance,
+      0
     )
 
     // Get SOL => USDC quote to estimate $USD cost
     const quoteUSD = yield* call(JupiterSingleton.getQuote, {
       inputTokenSymbol: 'SOL',
       outputTokenSymbol: 'USDC',
-      inputAmount: estimatedLamports.toNumber() / LAMPORTS_PER_SOL,
+      inputAmount: estimatedLamports / LAMPORTS_PER_SOL,
```

Comment:
Estimated lamports is implicitly converted to a `Number` here, we might want to make that explicit. Otherwise if we change `LAMPORTS_PER_SOL` to a bigint we're gonna lose all the decimals